### ISSUE TITLE
notificacion if battery temperature is high 

### ISSUE THREAD

**Issue by paulle** [dubrowgn/wattz#20]
What do you think about a notification popping up if the battery temperature is over a certain limit.
I believe this could be a very useful feature.
What do you think?

**Comment by dubrowgn** [comment]
I agree it could be a useful feature. In fact, the last several android devices I've owned already do/did this out of the box.

**Comment by paulle** [comment]
I've never noticed this, not at temperature of 40-50°C.

Gesendet von Proton Mail für Mobilgeräte

-------- Original-Nachricht --------
Am 10. Okt. 2022, 19:12, schrieb Dustin Brown :
